pages/_document.tsx

```diff
@@ -2,7 +2,7 @@ import { Html, Head, Main, NextScript } from "next/document";
 
 export default function Document() {
   return (
-    <Html>
+    <Html id="app">
```

Comment:
why was this needed?

Reply:
ah for the dark mode background, you could put the background classes on the outer div in the body instead